src/seedu/addressbook/AddressBook.java

```diff
@@ -1149,7 +1149,7 @@ private static String getUsageInfoForExitCommand() {
      * @return  string without the prefix
      */
     private static String removePrefix(String fullString, String prefix) {
-        return fullString.replace(prefix, "");
+        return fullString.replaceFirst(prefix, "");
```

Comment:
I see that you have already done the changes. Good!

When you modify the behavior, remember to update documentation and tests as well.